tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
```

Comment:
it'd be great if you can add a readme file in the tests directory to list the local testing steps (I'm assuming just install and run tox?). that's one of the things missed by newcomers to the project. @jimmc